.github/workflows/steam.yml

```diff
@@ -79,7 +82,8 @@ jobs:
 
             ASSETS="$(curl -s "${ASSETS_URL}")"
             WIN_ASSET_URL="$(jq -r '.[] | select(.name|test(".*x64.zip")) .browser_download_url' <<< ${ASSETS})"
-            MAC_ASSET_URL="$(jq -r '.[] | select(.name|test(".*.dmg")) .browser_download_url' <<< ${ASSETS})"
+            MAC_ASSET_URL="$(jq -r '.[] | select(.name|test(".*x86_64.dmg")) .browser_download_url' <<< ${ASSETS})"
+            MAC_ARM_ASSET_URL="$(jq -r '.[] | select(.name|test(".*arm64.dmg")) .browser_download_url' <<< ${ASSETS})"
```

Comment:
Needs adjusted for final names that are going to be used in releases giong forward.